hudi-client/src/main/java/org/apache/hudi/table/MarkerFiles.java

```diff
@@ -110,6 +158,10 @@ private String translateMarkerToDataPath(String markerPath) {
     return MarkerFiles.stripMarkerSuffix(rPath);
   }
 
+  public static String stripMarkerSuffix(String path) {
```

Comment:
nit: better to have the static methods right at the top?

Reply:
I can change it back, but is there a code style guide we are following ? I haven't seen this rule in google style guide for java atleast.

Reply:
this is not checkstyle enforced atm. So I am happy to let this be.